doc/admin-guide/files/records.config.en.rst

```diff
@@ -3485,6 +3485,26 @@ Client-Related Configuration
    engines. This setting assumes an absolute path.  An example config file is at
    :ts:git:`contrib/openssl/load_engine.cnf`.
 
+TLS v1.3 0-RTT Configuration
+----------------------------
+
+.. ts:cv:: CONFIG proxy.config.ssl.server.max_early_data INT 0
+
+   Specifies the maximum amount of early data in bytes that is permitted to be sent on a single connection.
+
+   Setting to ``0`` effectively disables 0-RTT.
```

Comment:
I also feel reasonably strongly that we should allow this to be configured in ssl_server_name.yaml, such that it can be disabled / enabled per TLS context.  I'm not sure if it makes sense to have max_early_data be overridable per context as well, maybe something to discuss.

Reply:
Sure, that sounds reasonable.

Reply:
I'd guess that we want them overridable too. eg, mycoolservice.com has a mix of services -- the remap for /static gets 0RTT, while /sillywebcounter shouldn't do 0rtt.

Reply:
Discussed with @shinrich and we think it's complicated enough to do a separate PR for this change.